src/main/java/ch/njol/skript/effects/EffToggleCanPickUpItems.java

```diff
@@ -15,14 +15,13 @@
 
 @Name("Toggle Picking Up Items")
 @Description("Determines whether living entities are able to pick up items or not")
-@Examples({
-	"forbid player from picking up items",
-	"send \"You can no longer pick up items!\" to player",
-	"",
-	"on drop:",
-		"\tif player can't pick	up items:",
-			"\t\tallow player to pick up items"
-})
+@Example("forbid player from picking up items")
+@Example("send \"You can no longer pick up items!\" to player")
+@Example("""
+	on drop:
+		if player can't pick	up items:
```

Comment:
```suggestion
		if player can't pick up items:
```
what happened here lmao

Reply:
Wacky, wonder why it didn't show up in the old code.